app/models/comment.rb

```diff
@@ -71,7 +71,7 @@ def check_for_spam
   validates :comment_content, uniqueness: {
     scope: [:commentable_id, :commentable_type, :name, :email, :pseud_id],
     unless: :is_deleted?,
-    message: ts("^This comment has already been left on this work. (It may not appear right away for performance reasons.)")
+    message: ts("^You’ve already left this comment here. (It may not appear right away for performance reasons.")
```

Comment:
It would be nice to internationalize this, by [overriding the default error message key](https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards#user-content-Overriding_a_default_error_message_key):

```suggestion
    message: :duplicate_comment
```
and then putting the translation, without the leading `^`, into [config/locales/models/en.yml](https://github.com/otwcode/otwarchive/blob/a0775cd551a588ee8e65bed69319a3b51ef069d6/config/locales/models/en.yml#L92)
```yml
  activerecord:
    errors:
      models:
        comment:
          attributes:
            comment_content:
              duplicate_comment: You've already ....
              format: "%{message}"
```
Could you also fix your typos? It should be a plain `'` in `You've` instead of a curly one and the closing parenthesis at the end should be there.